torch/nn/quantized/modules/conv.py

```diff
@@ -0,0 +1,109 @@
+# coding=utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import torch
+from torch._jit_internal import weak_module, weak_script_method
+from torch._ops import ops
+from torch.nn.modules.module import Module
+from torch.nn.modules.utils import _pair
+from torch.nn.quantized import functional as qF
+
+from torch.nn.modules.conv import _ConvNd
+
+"""Computes the output shape given convolution parameters."""
+def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
+                       output_padding=0):
+    return np.floor((input_size + 2 * padding - kernel_size - (kernel_size - 1)
+                    * (dilation - 1)) / stride) + 2 * output_padding + 1
+
+
+@weak_module
+class Conv2d(_ConvNd):
```

Comment:
should we change to common base class for nn._ConvNd and nnq._ConvNd as we discussed the other day?

Reply:
Good idea! Will do!

Reply:
Actually, now that I am thinking about it, I don't think we need a common base. Deleting of the parent variables works, and in the future we will not use the packed weights, so it should be fine. What do you think?